databuilder/query_engines/base_sql.py

```diff
@@ -273,6 +273,17 @@ def get_sql_element_no_cache(self, node: QueryNode) -> ClauseElement:
         # Fallback for unhandled types
         assert False, f"Unhandled query node type: {node!r}"
 
+    def get_sql_element_or_value(self, value: Any) -> Any:
+        """
+        Certain places in our Query Model support values which can either be QueryNodes
+        themselves (which require resolving to SQL) or plain static values (booleans,
+        integers, dates, list of dates etc) which we can return unchanged to SQLAlchemy.
+        """
+        if isinstance(value, QueryNode):
+            return self.get_sql_element(value)
+        else:
+            return value
+
```

Comment:
I think a rebase has gone wrong here, and this commit should actually now be a no-op.